tests/input/draft-template-old.xml

```diff
@@ -515,7 +515,18 @@ main(int argc, char *argv[])
 
       &RFC3552;
 
-      &I-D.narten-iana-considerations-rfc2434bis;
+      <reference anchor="I-D.narten-iana-considerations-rfc2434bis" target="https://www.ietf.org/archive/id/draft-narten-iana-considerations-rfc2434bis-09.txt" xml:base="https://bib.ietf.org/public/rfc/bibxml3/reference.I-D.narten-iana-considerations-rfc2434bis.xml" quote-title="true">
+        <front>
+          <title>Guidelines for Writing an IANA Considerations Section in RFCs</title>
+          <author fullname="Dr. Thomas Narten" initials="T." surname="Narten" />
```

Comment:
@rjsparks Note the salutation in the `fullname`. Not sure that's correct.

Reply:
We do not model honorifics separately. (Either in the datatracker or in the rfcxml). Some people have put them in the data for their full name in the datatracker.

Reply:
In this case, Do you want to leave that in expanded reference?

Reply:
Keep this as is on tests.